src/ert/storage/local_ensemble.py

```diff
@@ -217,8 +217,15 @@ def load_responses(
             if not input_path.exists():
                 raise KeyError(f"No response for key {key}, realization: {realization}")
             ds = xr.open_dataset(input_path, engine="scipy")
+            if "time" in ds.coords:
+                ds.coords["time"] = [
+                    datetime.fromisoformat(str(e.values).split(".", maxsplit=1)[0])
```

Comment:
Are we losing the possibility for millisecond accuracy here?

Reply:
Yes with this implementation we do. Do we really need milliseconds?  I think we might be able to also keep ms but nanoseconds are not possible

Reply:
Uncertain whether we "need" it, but Eclipse can output milliseconds. Whether `resdata` supports it I am not quite sure.

Reply:
Could we perhaps do the millisecond truncation in a more canonical way and not by string manipulation? Parsing the ISO string as is, and then set the millisecond part to zero using the datetime api?